src/listmode_buildblock/LmToProjData.cxx

```diff
@@ -683,11 +683,11 @@ process_data()
 			   {
 			     do_post_normalisation(bin);
 			 
-			     num_stored_events += event_increment;
+                 num_stored_events += bin.get_bin_value() *event_increment;
```

Comment:
This will break pre- and post-normalisation. In those case, the `bin_value` will be the "efficiency" factors. So counting those doesn't make sense to the user.

Also true for the `num_prompts_in_frame` etc.

It seems safer to leave this as-is, and just accept the fact that the reported values will be surprising when using bootstrapping (could let `LmToProjDataBootstrap` write a warning at the beginning).

Reply:
Mh that's right!